src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Ajuste configurações de cache
+description: Boost your edge application's efficiency with strategic cache handling with custom cache TTL values, object segmentation methods, and file slicing.
+meta_tags: build, cache, ttl, browser, edge, object
+namespace: docs_guides_build_tune_cache_settings
+permalink: /documentation/products/guides/build/tune-cache-settings/
+menu_namespace: buildMenu
+---
+
+Ajuste as [políticas de cache](/en/documentation/products/edge-application/cache-settings/) da sua edge application, incluindo valores de expiração do cache da edge ou do navegador, file slicing para segmentação de objetos maiores, e definição de cache key com base em query strings ou cookies.
+
+:::caution[warning]
+Algumas funcionalidades de cache requerem que o módulo [Application Acceleration](/en/documentation/products/edge-application/application-acceleration/) esteja ativo. Se o **Application Acceleration** estiver ativado, a transferência de dados pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/en/pricing/) para mais informações.
+:::
+
+## Via RTM
```

Comment:
```suggestion

---

## Via RTM
```

Reply:
Ai eu to deixando tudo sem o tracinho :(